virttest/utils_disk.py

```diff
@@ -835,6 +835,26 @@ def get_disk_by_serial(serial_str, session=None):
             return disk
 
 
+def new_disk_vol_name(pool_name):
+    """
+    According to BZ#1138523, the new volume name must be the next
+    created partition(sdb1, etc.), so we need to inspect the original
+    partitions of the disk then count the new partition number.
+
+    :param pool_name: Disk pool name
+    :return: New volume name or none
+    """
+    from virttest.libvirt_xml import pool_xml
```

Comment:
@balamuruhans We'd better really break the circular import if it is caused by method reorganize.

Reply:
@Yingshun I have taken care to fix it.